docs/models.yml

```diff
@@ -1505,56 +1500,85 @@ motion_poll:
     equal_fields: meeting_id
   meeting_id:
     type: relation
-    to: meeting/motion_poll_ids
+    to: meeting/poll_ids
 
-motion_option:
+option:
   id: number
+  weight:
+    type: number
+    default: 10000
+  text: HTMLStrict
   yes: decimal(6)
   no: decimal(6)
   abstain: decimal(6)
 
   poll_id:
     type: relation
-    to: motion_poll/option_ids
+    to: poll/option_ids
     equal_fields: meeting_id
+    required: true
   vote_ids:
     type: relation-list
-    to: motion_vote/option_id
+    to: vote/option_id
     on_delete: CASCADE
     equal_fields: meeting_id
+  content_object_id:
+    # TODO (this is the former user_id relation)
+    #type: relation
+    #to:
+    #  collection: user
+    #  field:
+    #    name: option_$_ids
+    #    type: structured-relation
+    #    replacement: meeting_id
+    type: generic-relation
+    to:
+      collection:  # Now, we have multiple models to vote about.
+        - motion
+        - assignment
+        - user  # This is problematic: Normally, all relations to user are structured...
```

Comment:
I'm confused. Why can you vote about an assignment? Is this a new feature? And, the more important question, why did we save the user in the option? We explicitly create the `assignment_candidate` models which we could use to solve this problem here, or not?

Reply:
> We explicitly create the assignment_candidate models which we could use to solve this problem here, or not?

No, assignment_candidates are used before an election - they are copied into the poll. You can have two polls in an assignment with different options <- This is why the users are in the options.

With a generic field, we have the possiblity to vote about other thinks if we want to.